### ISSUE TITLE
Check if car is EV or not to avoid log clutter

### ISSUE THREAD

**Issue by lmaced0** [athal7/homebridge-hyundai-bluelink#266]
**Is your feature request related to a problem? Please describe:**

Yes. If your car is not an EV, your Homebridge logs will be cluttered with  this message:
<img width="837" alt="Screenshot 2022-10-02 at 2 49 54 PM" src="https://user-images.githubusercontent.com/83371481/193475786-3096f681-6f03-4e6e-85ae-62ba08cc25de.png">

**Describe the solution you'd like:**
It would be nice to have a setting that indicates if car is an EV or not a filter that.

**Comment by jm3** [comment]
Yes, please, it's misleading and messy to be emitting dozens of repeated log lines that an ICE vehicle isn't charging 🤣
